frontends/infinite-corridor/src/pages/FieldPageSkeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { BannerPage } from "ol-util"
+import Container from "@mui/material/Container"
+import { useFieldDetails } from "../api/fields"
+
+interface FieldSkeletonProps {
+  children: React.ReactNode
+  name: string
+}
+
+/**
+ * Common structure for field-oriented pages.
+ *
+ * Renders the field title and avatar in a banner.
+ */
+const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
+  children,
+  name
+}) => {
+  const field = useFieldDetails(name)
+
+  return (
+    <BannerPage
+      compactOnMobile
+      bannerContent={
+        <Container className="field-title-container">
+          <div className="field-title-row">
+            <img alt={field.data?.title} src={field.data?.avatar_small ?? ""} />
```

Comment:
The medium avatar is used for discussion channel avatars in the header ([example](https://discussions-rc.odl.mit.edu/c/mbchannel02)), while the small avatar is used for the left side vertical channel list.  So I'm guessing  the medium avatar should be used for field channel banners too.

<img width="435" alt="Screen Shot 2022-07-26 at 3 47 17 PM" src="https://user-images.githubusercontent.com/187676/181099139-d1aeec5f-8e70-475e-94b7-a9e4ca4314a4.png">